### ISSUE TITLE
this plugin is didn't work

### ISSUE THREAD

**Issue by wn0x00** [LiHowe/vuepress-plugin-mermaid#1]
i can't make it work

**Comment by LiHowe** [comment]
Can you provide a usage scenario or some error screen capture？Thx

**Comment by azat-io** [comment]
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/5698350/188230087-8b4bce2d-01e4-40ec-b98c-17e6aa18ef73.png">

**Comment by snailwei** [comment]
Dose not work for `"vuepress": "^2.0.0-beta.60"`

**Comment by azat-io** [comment]
@sweihub You can try to use [vuepress-plugin-mermaid-wrapper](https://github.com/azat-io/azat-io/tree/main/plugins/mermaid-wrapper)

**Comment by LiHowe** [comment]
solved the issue with [PR#5](https://github.com/LiHowe/vuepress-plugin-mermaid/pull/5)

**Comment by snailwei** [comment]
> @sweihub You can try to use [vuepress-plugin-mermaid-wrapper](https://github.com/azat-io/azat-io/tree/main/plugins/mermaid-wrapper)

The memaid-wrapper not working with `"vuepress": "^2.0.0-beta.60"` as well, thanks anyway, I created a blank VuePress 2 project, it failes to load the plugin.

**Comment by snailwei** [comment]
> solved the issue with [PR#5](https://github.com/LiHowe/vuepress-plugin-mermaid/pull/5)

Thanks, did you push to the npm repository, I still cannot get it to work for VuePress 2
- `"vuepress-plugin-mermaid-next": "^0.1.14"`
- `"vuepress": "^2.0.0-beta.60"`
- node version v14.18.0 / v19.4.0
```
Uncaught SyntaxError: The requested module '/@fs/home/swei/docs2/node_modules/moment-mini/moment.min.js?v=51b21672' does not provide an export named 'default' (at mermaid.core.mjs?v=51b21672:7:8)
```
